### PR TITLE
Add scmRepo when create pipeline and remove refreshScmRepo

### PR DESCRIPTION
**Pull request by minzcmu**
- Add `scmRepo` blob when creating pipeline
- Remove `refreshScmRepo`  in `sync` 
- Because we do not have`scmUri` update endpoint now, no need to refresh the `scmRepo`

### CODE REVIEW

**Review by tkyi — APPROVED** [review]

File: lib/pipeline.js [hunk 1]
```diff
@@ -38,10 +38,8 @@ class PipelineModel extends BaseModel {
 
         const factory = JobFactory.getInstance();
 
-        // update the generated scm repo
-        return this.refreshScmRepo()
-            // get the pipeline configuration
-            .then(() => this.getConfiguration())
+        // get the pipeline configuration
+        return this.getConfiguration()
```
Comment (petey) [inline lib/pipeline.js:64]: getConfiguration should appear in source order before it is used.

File: lib/pipelineFactory.js [hunk 2]
```diff
@@ -34,21 +34,29 @@ class PipelineFactory extends BaseFactory {
...
-            modelConfig.createTime = (new Date()).toISOString();
+        const userFactory = UserFactory.getInstance();
+        const modelConfig = config;
 
-            return super.create(modelConfig);
-        });
+        modelConfig.createTime = (new Date()).toISOString();
+
+        return userFactory.get({ username: Object.keys(config.admins)[0] })
+                .then(user => user.unsealToken())
```
Comment (petey) [inline lib/pipelineFactory.js]: Why is this double indented?

**Review by petey — CHANGES_REQUESTED** [review]

File: lib/pipeline.js (hunk 1 above)
Comment (minzcmu) [reply lib/pipeline.js:64]: fixed :)

**Review by petey — APPROVED** [review]
going_right_round

**Pull request merged**